types/inspect/inspect.d.tl

```diff
@@ -0,0 +1,10 @@
+global InspectOptions = record
```

Comment:
Is there any way to avoid the global here?

Reply:
not at this point, unfortunately, because we don't really have a way to declare that it returns a table with a `__call` metamethod, but this should be an okay approximation of the declaration now.